setup.py

```diff
@@ -37,7 +37,7 @@
     include_package_data=True,
     install_requires=[
         "python-dotenv",
-        "Click>=7.1,<8",
+        "Click>=7.1,<=8.1",
```

Comment:
Please use <9, as we expect any 8.x version to be OK. The current `<8` is meant to say "any future compatible version". If we are compatible with 8.1 now, with your changes, we should be good to go until 9.x.